packages/server/lib/browsers/cdp_automation.ts

```diff
@@ -83,33 +86,37 @@ export const CdpAutomation = (sendDebuggerCommandFn: SendDebuggerCommand) => {
     // see MakeCookieFromProtocolValues for information on how this cookie data will be parsed
     // @see https://cs.chromium.org/chromium/src/content/browser/devtools/protocol/network_handler.cc?l=246&rcl=786a9194459684dc7a6fded9cabfc0c9b9b37174
 
-    _.defaults(cookie, {
-      name: '',
-      value: '',
+    const setCookieRequest: cdp.Network.SetCookieRequest = _({
```

Comment:
What does `_({...})` actually do here? The underscore docs suggest this is "OOP style", but if we're not chaining off of it, what is this doing if not just a plain object construct? (Does `_` do something useful like removing undefineds, maybe?)